cwr/api_test.go

```diff
@@ -217,6 +261,22 @@ func testTxRecords(x *testIndex,
 					}
 				}
 			}
+			for _, swrCid := range tx["DetailRecords"].(map[string]interface{})["SWR"].([]interface{}) {
+				obj, err := x.store.Get(swrCid.(*cid.Cid))
+				if err != nil {
+					return err
+				}
+				if err := obj.Decode(record); err != nil {
+					return err
+				}
+				if record.RecordType == "SWR" {
+					if record.WriterFirstName != "" {
+						if err := assertQuerySWR(record, `{ writer_control(writer_first_name:%q) { writer_first_name } }`, record.WriterFirstName); err != nil {
```

Comment:
We should test a top level query like getting a works and all its contributors in one go.

Reply:
we have it on line 244 though might need to add all fields

Reply:
update at 7d2774d